package.json

```
@@ -54,7 +54,7 @@
     "@docusaurus/plugin-svgr": "^3.7.0",
     "@docusaurus/theme-classic": "^3.7.0",
     "@docusaurus/tsconfig": "^3.7.0",
-    "@inkeep/widgets": "^0.2.292",
+    "@inkeep/cxkit-react": "^0.5.31",
```

Just bumped a new version to fix the previews.
```suggestion
    "@inkeep/cxkit-react": "^0.5.32",
```